Apps/phgsgmail/gsgmail.json

```diff
@@ -111,13 +107,83 @@
             "description": "Contents of Service Account JSON file",
             "data_type": "password",
             "order": 1
+        },
+        "label": {
+            "description": "Mailbox Label (folder) to be polled",
+            "data_type": "string",
+            "value_list": [],
+            "default": "",
+            "order": 2
+        },
+        "ingest_manner": {
+            "description": "How to ingest",
+            "data_type": "string",
+            "required": true,
```

Comment:
To support backward compatibility, remove _"required": true_ and use the default `oldest first` in JSON and code